src/main/java/model/Car.java

```diff
@@ -0,0 +1,61 @@
+package model;
+
+public class Car {
+    private static final int DRIVE_POSITION = 1;
+    private static final int CONDITION_MIN_VALUE = 0;
+    private static final int CONDITION_MAX_VALUE = 10;
+    private static final int CONDITION_GOOD_MIN_VALUE = 4;
+    private static final boolean GOOD = true;
+    private static final boolean BAD = false;
+    private static final int CORRECT_NAME_LENGTH_LIMIT = 5;
+    private String name;
+    private int position = 0;
+
+    public Car(String name) {
+        if (isCorrectName(name)) {
+            this.name = name;
+            return;
+        }
+        throw new RuntimeException("부적합한 입력.");
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    private boolean isCorrectName(String name) {
+        return isNotBlank(name) && isLengthIsBelowCorrectNameLengthLimit(name);
+    }
+
+    private boolean isNotBlank(String name) {
+        return !name.trim().isEmpty() && !name.contains(" ");
```

Comment:
```suggestion
        return !name.trim().isEmpty();
```
``name.trim().isEmpty()`` 와  ``name.contains(" ")`` 이 둘은 중복된 로직입니다. 중복을 제거하는게 좋을 것 같습니다.

Reply:
놓친 부분인 것 같습니다. 지적 감사합니다.